erpnext/stock/doctype/purchase_receipt/test_purchase_receipt.py

```diff
@@ -2017,6 +2017,47 @@ def test_purchase_receipt_with_backdated_landed_cost_voucher(self):
 		ste7.reload()
 		self.assertEqual(ste7.items[0].valuation_rate, valuation_rate)
 
+	def test_purchase_receipt_provisional_accounting(self):
+		# Step - 1: Create Supplier
+		from erpnext.buying.doctype.supplier.test_supplier import create_supplier
+
+		supplier = create_supplier(default_currency="USD")
+
+		# Step - 2: Create Company with Provisional Accounting
```

Comment:
@s-aga-r Can we try and avoid new company creations, new company creation takes a lot of time and increases the test running time. There are already tests related to this on existing company, maybe just reuse that instead of creating a new company

Reply:
Adding case here: https://github.com/s-aga-r/erpnext/blob/9d27a25e5f2335386402f96f83a10729695b20c8/erpnext/accounts/doctype/purchase_invoice/test_purchase_invoice.py#L1469-L1526